Adicionada a opção de exclusão

BranBadures opened the pull request:
Adicionei na view table-row-headers mais uma seção para comportar o botão de exclusão e, por conseguinte, adicionei a opção de exclusão de registros na view fields.blade

BranBadures committed:
Adicionada a opção de exclusão

Adicionei na view table-row-headers mais uma seção para comportar o botão de exclusão e, por conseguinte, adicionei a opção de exclusão de registros na view fields.blade

thiagogomesverissimo commented:
Muito bom. 
Só um detalhe que precisa corrigir. Para mostrar o icone de lixeira use o fontawesome:
https://fontawesome.com/search?q=trash&s=solid%2Cbrands
Isto é:
<i class="fa-solid fa-trash"></i>

BranBadures committed:
Conjunto de alterações #1

-Edição do botão de remoção
-Edição da tabela na index
-Alteração na tabelação das páginas

ricardfo commented:
O TermoController está com conflito;
No TermoController no método "store" têm um update logo após um create, não entendi a atualização logo após criar um termo. Também têm um dd  e um request flash comentado, devem ser excluídos do pull-request já que não são mais utilizados.

BranBadures committed:
Conjunto de alterações 2
-Refeito o sistema de pesquisas
-Alterado a view para comportar a nova pesquisa
-Refatoração da TermoController

BranBadures committed:
Resolvido conflito de versão do index do TermoController
